src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue5669.cs

```diff
@@ -19,8 +19,8 @@ public void SearchBarMaxLength()
 			App.EnterText("SearchBar", "r");
 			App.Click("ChangeValue");
 			App.EnterText("SearchBar", "r");
-			App.EnterText("SearchBar", "c");
-#if MACCATALYST
+			App.EnterText("SearchBar", "c");			
+#if MACCATALYST 
```

Comment:
super nit, revert this ws change

Reply:
@mattleibow , I have updated the code. please let me know if you have any concerns.